src/app/interface-form/interface-form.component.ts

```diff
@@ -18,4 +18,8 @@ export class InterfaceFormComponent {
       Validators.max(this.userDeposit),
     ]),
   });
+
+  onSubmit() {
```

Comment:
Don't forget to set a type of the returned value (in this case - void)

Reply:
done